scriptmodules/emulators/dosbox.sh

```diff
@@ -98,24 +104,25 @@ midi_synth start
 "$md_inst/bin/dosbox" "\${params[@]}"
 midi_synth stop
 _EOF_
-    chmod +x "$romdir/pc/+Start DOSBox.sh"
-    chown $user:$user "$romdir/pc/+Start DOSBox.sh"
-
-    local config_path=$(su "$user" -c "\"$md_inst/bin/dosbox\" -printconf")
-    if [[ -f "$config_path" ]]; then
-        iniConfig "=" "" "$config_path"
-        iniSet "usescancodes" "false"
-        iniSet "core" "dynamic"
-        iniSet "cycles" "max"
-        iniSet "scaler" "none"
-        if isPlatform "rpi" || [[ -n "$(aconnect -o | grep -e TiMidity -e FluidSynth)" ]]; then
-            iniSet "mididevice" "alsa"
-            iniSet "midiconfig" "128:0"
+        chmod +x "$romdir/pc/$launcher_name"
+        chown $user:$user "$romdir/pc/$launcher_name"
+
+        local config_path=$(su "$user" -c "\"$md_inst/bin/dosbox\" -printconf")
+        if [[ -f "$config_path" ]]; then
+            iniConfig "=" "" "$config_path"
+            iniSet "usescancodes" "false"
+            iniSet "core" "dynamic"
+            iniSet "cycles" "max"
+            iniSet "scaler" "none"
+            if isPlatform "rpi" || [[ -n "$(aconnect -o | grep -e TiMidity -e FluidSynth)" ]]; then
```

Comment:
unrelated to this changeset, but what's the logic here with checking for rpi or the output of aconnect ? Since the depends only installs timidity for the rpi etc.

Reply:
Also the fact there is a check for this with variable needs_synth ?

Reply:
ignore that - that's in the launch script. sorry.

Reply:
We set our daemon only on Pi for now, as it might be risky to enable on platforms that don't have some kind of working ALSA mixing (dmix or hardware mixing). However, the check above is to configure the ALSA midi port if there is an existing daemon running that's installed by the user (e.g. timidity-daemon).